packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -442,6 +464,13 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
         this.disposeCore(true, error);
     };
 
+    protected documentDisconnectHandler = (error: IFluidErrorBase & OdspError, clientId: string) => {
+        if (clientId === this.clientId) {
+            this.logger.sendTelemetryEvent({ eventName: "DocumentDisconnect", clientId: this.clientId }, error);
+            this.disposeCore(true, error, true);
```

Comment:
can you please add comments to boolean args on their meaning? Something like
true, // bla-bla arg